docs/dictionary/en-custom.txt

```diff
@@ -228,6 +230,7 @@ nic
 nigzpbgugpsavdmfyl
 nlcggvjgnsdxn
 nmstate
+nmstatectl
```

Comment:
not sure this is needed either? Or is it coming from the addition of devscripts readme in the doc?

Reply:
Thank you. I may have modified observing some other failure.